src/main.c

```diff
@@ -132,6 +132,9 @@ int main (int argc, char* argv[])
 
 	//Remove PID file
 	removepid();
-	logg("########## FTL terminated after %e s! ##########", 1e-3*timer_elapsed_msec(EXIT_TIMER));
+
+	char buffer[42] = { 0 };
+	format_time(buffer, 0, timer_elapsed_msec(EXIT_TIMER));
+	logg("########## FTL terminated after%s! ##########", buffer);
```

Comment:
```suggestion
	logg("########## FTL terminated after %s! ##########", buffer);
```
Is the space missing or does the variable already contain the space?

Reply:
It already contains the space: https://github.com/pi-hole/FTL/pull/907/files#diff-46b881d73b3e6407e73cdcc59cdaf1de4bb5a0122acf867cb88bdd28ca34ca45R202